crates/aiken-project/src/error.rs

```diff
@@ -74,7 +74,7 @@ pub enum Error {
         error: Box<ParseError>,
     },
 
-    #[error("Checking")]
```

Comment:
I was generically calling this checking because some checks/errors aren't necessarily type related. I'm fine with which ever FWIW.

Reply:
Yeah, I tried different things here actually; Even went for full sentences like "Analyzing modules types and semantics". But nothing really pleased me much; felt too heavy. I thought "Type-checking" was slightly more informative, even though as you say, it's not only about type checking. 

This line is really just a title / label, so it doesn't matter much; we could perhaps go for something a little _funny_ 🤷 ... 

```
  × Type-checking
  ╰─▶ Unexpected labeled argument 't'
```

vs 

```
  × While trying to make sense of your code...
  ╰─▶ I tripped over an unexpected labeled argument 't'
```